kitsune/sumo/static/sumo/js/tests/instant_search_tests.js

```diff
@@ -0,0 +1,140 @@
+import {default as mochaJsdom, rerequire} from 'mocha-jsdom';
+import {default as chai, expect} from 'chai';
+import React from 'react';
+import chaiLint from 'chai-lint';
+import sinon from 'sinon';
+
+import mochaK from './fixtures/mochaK.js';
+import mochaJquery from './fixtures/mochaJquery.js';
+import mochaGoogleAnalytics from './fixtures/mochaGoogleAnalytics.js';
+import mochaNunjucks from './fixtures/mochaNunjucks.js';
+import mochaGettext from './fixtures/mochaGettext.js';
+
+chai.use(chaiLint);
+
+describe('instant search', () => {
+  mochaJsdom({useEach: true});
+  mochaJquery();
+  mochaK();
+  mochaGoogleAnalytics();
+  mochaGettext();
+  mochaNunjucks();
+  /* globals window, document, $ */
+
+  describe('', () => {
```

Comment:
The describe blocks usually go something like 

```js
describe('library', () => {
  describe('method in library', () => {
    it('should do the things');
  });
});
```

There isn't really a library here, so that second layer doesn't make a lot of sense, but it is still needed for the `mocha*` helpers. At least, I think it is. I remember going through a lot of debugging here, so maybe this is just a spurious change.